loopchain/channel/channel_service.py

```diff
@@ -333,10 +333,8 @@ def switch_role(self):
 
     async def __init_peer_auth(self):
         try:
-            node_key: bytes = await StubCollection().peer_stub.async_task().get_node_key(ChannelProperty().name)
+            node_key: bytes = await StubCollection().peer_stub.async_task().get_node_key()
             self.__peer_auth = Signer.from_prikey(node_key)
-        except KeyError:
-            self.__peer_auth = Signer.from_channel(ChannelProperty().name)
         except Exception as e:
             logging.exception(f"peer auth init fail cause : {e}")
```

Comment:
There is logging.exception in exit_and_msg also.

Reply:
Resolved :)